### PR TITLE
fix: es-db client pooling

### PR DESCRIPTION
**Pull request by devadvocado**

### CODE REVIEW

File: api/app/elastic.js [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
 const elasticsearch = require('elasticsearch')
 
-const createClient = async () => {
-    return elasticsearch.Client({
-        hosts: [(process.env.ES_HOST || `http://localhost:9200`)],
-        maxRetries: 30,
-        requestTimeout: 30000
-    })
-}
+const client = elasticsearch.Client({
```
Comment (jakubdyszkiewicz) [inline api/app/elastic.js]: What is the default connections limit of the connection pool?

**Comment by jakubdyszkiewicz** [comment]
Now it's super easy to kill application. See the issue https://github.com/Kong/kuma-demo/issues/43

> As far as I understand, it will get us back into the situation where changes to TrafficPermissions won't take effect for 30 more seconds (while existing connections are being drained)

1) We could adjust draining time for the demo deployment
2) Isn't draining with HTTP more inteligent (GOAWAY with HTTP2 and `Connection: close` with HTTP/1.1)?

File: api/app/elastic.js (hunk 1 above)
Comment (devadvocado) [reply api/app/elastic.js]: @jakubdyszkiewicz dont see any reference of the limit on the docs

**Comment by jakubdyszkiewicz** [comment]
Why was it closed?

**Pull request closed without merging**